data/brands/shop/supermarket.json

```
@@ -9627,6 +9641,21 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "יוחננוף",
+      "locationSet": {"include": ["il"]},
+      "tags": {
+        "brand": "יוחננוף",
+        "brand:en": "Yochananof",
+        "brand:he": "יוחננוף",
+        "brand:wikidata": "Q25492676",
+        "brand:wikipedia": "he:רשת יוחננוף",
+        "name": "יוחננוף",
```

```suggestion
        "brand:wikidata": "Q25492676",
        "name": "יוחננוף",
```
NSI no longer includes `brand:wikipedia`, see  #6481.